cswinrt/strings/WinRT_Interop.cs

```diff
@@ -17,12 +17,11 @@ namespace WinRT.Interop
 {
     // IActivationFactory
     [Guid("00000035-0000-0000-C000-000000000046")]
-    internal struct IActivationFactoryVftbl
+    internal unsafe struct IActivationFactoryVftbl
     {
-        public unsafe delegate int _ActivateInstance(IntPtr pThis, out IntPtr instance);
-
         public IInspectable.Vftbl IInspectableVftbl;
```

Comment:
I am still seeing this pattern. As previously mentioned the fully types vtable does introduce a fair bit of additional metadata overhead with little real value. The previous suggestion was to use a `void*[]` and index/cast at the dispatch site. The size reduction would be substantial in some instances. Is this something we just don't want to do in this PR?

Reply:
We can't just use `void*[]` for the vtable unless we want to re-allocate a delegate every time we have to call a generic method. We would need to cache those somewhere. I'm fine with moving to a caching mechanism for those and using pointer indirection for non-generic methods in the future, but I'd rather not move to that in this PR given its size.

Reply:
We shouldn't require a delegate allocation for this with function pointers right?

Regardless of the answer I completely agree this PR is big enough. Let's continue this conversation during the perf/optimization pass.

Reply:
We don't need a delegate allocation for IActivationFactory, that is true. We only need delegate allocations for methods with generic parameters or return values on WinRT interfaces.